pkg/epforwarder/epforwarder.go

```diff
@@ -111,35 +117,29 @@ type passthroughPipeline struct {
 }
 
 type passthroughPipelineDesc struct {
-	eventType              string
-	endpointsConfigPrefix  string
-	hostnameEndpointPrefix string
+	eventType                     string
+	endpointsConfigPrefix         string
+	hostnameEndpointPrefix        string
+	defaultBatchMaxConcurrentSend int
+	defaultBatchMaxContentSize    int
 }
 
 // newHTTPPassthroughPipeline creates a new HTTP-only event platform pipeline that sends messages directly to intake
 // without any of the processing that exists in regular logs pipelines.
 func newHTTPPassthroughPipeline(desc passthroughPipelineDesc, destinationsContext *client.DestinationsContext) (p *passthroughPipeline, err error) {
-	configKeys := config.LogsConfigKeys{
-		CompressionLevel:        desc.endpointsConfigPrefix + ".compression_level",
-		ConnectionResetInterval: desc.endpointsConfigPrefix + ".connection_reset_interval",
-		LogsDDURL:               desc.endpointsConfigPrefix + ".logs_dd_url",
-		DDURL:                   desc.endpointsConfigPrefix + ".dd_url",
-		DevModeNoSSL:            desc.endpointsConfigPrefix + ".dev_mode_no_ssl",
-		AdditionalEndpoints:     desc.endpointsConfigPrefix + ".additional_endpoints",
-		BatchWait:               desc.endpointsConfigPrefix + ".batch_wait",
-		BatchMaxConcurrentSend:  desc.endpointsConfigPrefix + ".batch_max_concurrent_send",
-	}
-	endpoints, err := config.BuildHTTPEndpointsWithConfig(configKeys, desc.hostnameEndpointPrefix)
+	endpoints, err := config.BuildHTTPEndpointsWithConfig(config.NewLogsConfigKeys(desc.endpointsConfigPrefix), desc.hostnameEndpointPrefix)
 	if err != nil {
 		return nil, err
 	}
 	if !endpoints.UseHTTP {
 		return nil, fmt.Errorf("endpoints must be http")
 	}
-	// since some of these pipelines can be potentially very high throughput we increase the default batch send concurrency
-	// to ensure they are able to support 1000s of events per second
+	// epforwarder pipelines apply their own defaults on top of the hardcoded logs defaults
 	if endpoints.BatchMaxConcurrentSend <= 0 {
-		endpoints.BatchMaxConcurrentSend = 10
+		endpoints.BatchMaxConcurrentSend = desc.defaultBatchMaxConcurrentSend
+	}
+	if endpoints.BatchMaxContentSize <= pkgconfig.DefaultBatchMaxContentSize {
+		endpoints.BatchMaxContentSize = desc.defaultBatchMaxContentSize
```

Comment:
Yeah this is a bit confusing because there's multiple levels of defaults. 

There's no good reason (so far) to go lower than the hardcoded defaults, so the way this config is setup is:

* if a custom config value is specified which is *greater* than the hardcoded defaults then use that 
* else, use the epforwarder pipeline specific default 

Does that make sense?